content/en/docs/concepts/storage/volumes.md

```diff
@@ -255,11 +255,40 @@ If that is filled up from another source (for example, log files or image
 overlays), the `emptyDir` may run out of capacity before this limit.
 
 {{< note >}}
-If the `SizeMemoryBackedVolumes` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled,
-you can specify a size for memory backed volumes.  If no size is specified, memory
+The `SizeMemoryBackedVolumes` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled by default since v1.22,
+so you can specify a size for memory backed volumes.  If no size is specified, memory
 backed volumes are sized to node allocatable memory.
 {{< /note>}}
 
+{{< caution >}}
+If the user does not specify a `sizeLimit` for `emptyDir`, user can create a
+memory backed `emptyDir` with node allocatable size unintentionally.  Also,
+user can create multiple memory backed `emptyDir`s on the same node, each set
+to a node allocatable size.  And what Kubernetes sees as memory usage is the
+amount that is actually consuming the memory　backed `emptyDir`, so even after
+you create a memory-backed `emptyDir` with node allocatable size, the other Pods
+can be scheduled to the same node.  As a result, multiple memory backed
+`emptyDir`s with node allocatable size can be created on the same node.
+
```

Comment:
Consider ending the `caution` block here, with some advise to continue reading the page to learn  about managing those risks.

Also, how about moving the `caution` earlier in the page (eg: around line 224)?

Reply:
I think an explanation about the whole emptyDir is needed before talking about memory backed volume. Instead of moving `caution` block earlier, I made a subsection about memory backed emptyDir, what do you think?